### PR TITLE
mod_acl_user_groups: Insert default ACL rules

### PR DESCRIPTION
**Pull request by ddeboer**
See #1131.

### CODE REVIEW

File: modules/mod_acl_user_groups/mod_acl_user_groups.erl [hunk 1]
```diff
@@ -601,19 +601,22 @@ manage_datamodel(Context) ->
...
-                    {hascollabmanager, 
+                    {hascollabmanager,
                         [{title, {trans, [{en, <<"Manager">>},{nl, <<"Beheerder">>}]}}],
                         [{acl_collaboration_group, person}]
                     }
-                ]
+                ],
+            data = [
+                {acl_rules, acl_default_rules:get_default_rules()}
+            ]
```
Comment (ddeboer) [inline modules/mod_acl_user_groups/mod_acl_user_groups.erl]: This is the major change. Sorry about the large diff, but my IDE (correctly) strips all trailing spaces.

Comment (arjan) [inline modules/mod_acl_user_groups/mod_acl_user_groups.erl]: Aren't rules that are inserted programmatically causing that the rules cannot be edited in the UI anymore?

Comment (ddeboer) [inline modules/mod_acl_user_groups/mod_acl_user_groups.erl]: Correct. That’s why I’m keeping the set of default rules to the bare minimum, hopefully covering the 80%. Do you nonetheless see some problems with it?

Of course, if you really do not want these default rules, you can always `observe_manage_data(#manage_data{module = Module, props = {acl_rules, Rules}}, Context)` in your site and get rid of them or change them.

**Comment by mworrell** [comment]
I am ok with this.  @fredpook and I are planning to add a `is_disabled` flag to the ACL rules, so that ACL rules can be temporarily disabled, which is very useful for schema-installed rules.

**Comment by mworrell** [comment]
Also merged 25a3da9 into 0.x

**Comment by ddeboer** [comment]
Thanks, @mworrell! 

@fredpook @mworrell Please create an issue for `is_disabled` so we can discuss whether it’s needed.

**Pull request merged**